pkgs/applications/version-management/git-and-tools/bump2version/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, isPy27, pytest, testfixtures, lib }:
+
+buildPythonPackage rec {
+  pname = "bump2version";
+  version = "1.0.0";
+  disabled = isPy27;
+
+  src = builtins.fetchGit {
+    url = "https://github.com/c4urself/${pname}.git";
+    ref = "refs/tags/v${version}";
+  };
+
+  # Remove tests that calls bumpversion that is not available at this point.
+  # That tests work in `nix-shell -f $NIXPKGS -iA python3Packages.bump2version`
+  patches = [ ./nix_test.patch ]; 
+
+  checkInputs = [ pytest testfixtures ];
+  # X's in pytest are git tests which won't run in sandbox
+  checkPhase = ''
+    pytest tests/
```

instead of the patch, you can just disable through pytest
```suggestion
    pytest tests/ -k 'not usage_string_fork'
```